xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1124,7 +1124,24 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
   if (ParameterNotNull(parameterObject, "track"))
     details.m_iTrack = (int)parameterObject["track"].asInteger();
   if (ParameterNotNull(parameterObject, "rating"))
-    details.m_fRating = parameterObject["rating"].asFloat();
+  {
+    details.SetRating(parameterObject["rating"].asFloat());
+    if (ParameterNotNull(parameterObject, "votes"))
+      details.SetVotes(StringUtils::ReturnDigits(parameterObject["votes"].asString()));
+    updatedDetails.insert("ratings");
+  }
+  if (ParameterNotNull(parameterObject, "ratings"))
+  {
+    CVariant ratings = parameterObject["ratings"];
+    for (auto& rIt = ratings.begin_map(); rIt != ratings.end_map(); rIt++)
```

Comment:
The other platform compilers don't like this :(
http://jenkins.kodi.tv/job/LINUX-RBPI/5305/console